dev/archery/archery/crossbow/cli.py

```diff
@@ -551,3 +551,42 @@ def upload_artifacts(obj, tag, sha, patterns, method):
     queue.github_overwrite_release_assets(
         tag_name=tag, target_commitish=sha, method=method, patterns=patterns
     )
+
+
+@crossbow.command()
+@click.option('--dry-run/--execute', default=False,
+              help='Just display process, don\'t download anything')
+@click.option('--days', default=90,
+              help='Branches older than this amount of days will be deleted')
+@click.pass_obj
+def delete_old_branches(obj, dry_run, days):
+    """
+    Deletes branches on queue repository (crossbow) that are older than number
+    of days.
```

Comment:
I would add a caveat here that deleting to many branches at once can trigger githubs hacking/fraud protection, though I do not know what the threshold is for that.

Reply:
Good idea, I've added a new CLI argument to limit the maximum number of branches to delete.